stable/rabbitmq/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 6.8.5
+version: 6.9.0
```

A new minor version appears in master, in this case, you need to bump the minor version again:
```suggestion
version: 6.10.0
```